server/src/services/blob_storage/filesystem_impl.ts

```diff
@@ -170,4 +170,25 @@ export default class Filesystem implements BlobStorage {
         this._directory = directory;
         this._isWindows = isWindows;
     }
+
+    renameFile(f: File): Promise<Result<boolean>> {
+        try {
+            // eslint-disable-next-line security/detect-non-literal-fs-filename -- names are generated by the server
```

Comment:
maybe provide a lil more context here- something like "typescript wants to guard against malicious file renaming, but since the re-name is generated server-side and not by the end user, there is no security risk"

Reply:
I'll see how to leave a multi-line es-lint disable, because I don't want the line to get mega-long